package.json

```diff
@@ -10,10 +10,11 @@
         "lint:css": "stylelint './src/**/*.css' && prettier './src/**/*.css' --check",
         "format": "d2-style apply && yarn format:css",
         "format:css": "prettier './src/**/*.css' --write",
-        "cypress:open": "d2-utils-cypress open --appStart 'yarn start'",
-        "cypress:run": "d2-utils-cypress run --appStart 'yarn start'",
-        "cypress:capture": "d2-utils-cypress run --capture --appStart 'yarn start' --serverMinorVersion 36",
-        "cypress:stub": "d2-utils-cypress run --stub --appStart 'yarn start' --serverMinorVersion 36"
+        "cypress:start": "BROWSER=none yarn start",
+        "cypress:open": "wait-on 'http://localhost:3000' && cypress open",
+        "cypress:run": "wait-on 'http://localhost:3000' && cypress run",
```

Comment:
@HendrikThePendric, @varl:
  
To me it's a little unclear how we expect users to use these commands locally. Normally `yarn cypress:open` would start the app and then open cypress, whereas now it expects the app to be started separately.

I think aligning the cypress:open and cypress:run scripts with the capture and stub scripts is more consistent (so starting the app for open and run as well).

Reply:
Ah I see they're actually used in the stub and capture scripts. So maybe then adding a separate open script for local usage would make sense? Which also starts the app, but runs live, say `cypress:live`?

Reply:
I've added the `cypress:live` script, but this led to a couple of other changes. The scripting we were using for capture and stub didn't work for the `live` script as they wouldn't exit when you'd exit cypress. So you'd have to exit the process in the shell yourself, alongside closing the cypress GUI.

With npm I'd use lifecycle scripts for this (pre, post) but those aren't available in yarn. So I've used [start-server-and-test](https://github.com/bahmutov/start-server-and-test), which is from a cypress maintainer, and recommended in their docs: https://docs.cypress.io/guides/continuous-integration/introduction#Boot-your-server.

Now when running `cypress:open`, when you're done you can close the gui and it'll exit the script and kill the server. I simplified the scripts a little, let me know if you hate it.

Reply:
I don't care that much, but you could have added `--kill-others` and `--success first` to the concurrently script to do the same thing.

```sh
concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture' --kill-others --success first

concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub' --kill-others --success first
```

Reply:
Ah I see, didn't know that. Yeah I thought I'd go with the one from the cypress docs as the maintainer (bahmutov) mentions correctly handling things like killing the server process etc. Seemed nice and readable and when it worked I didn't look any further.

Plus the maintainer is a ninja.